common-practices-tools/contribution/contrib-first.md

```diff
@@ -0,0 +1,46 @@
+# Contrib First
```

Comment:
Not sure quite where this goes, but we should be ensuring we have approval from clients before we contribute code, including talking about security, license and boilerplate the client may need (especially for new projects).
In many cases there are contract terms that cover this and we have or can get blanket approvals for various types of contribution.

Reply:
Hmm maybe a sentence or two in the top section.  I'll take a whack at it.

Reply:
I added this concept to the top.